lib/astarte/client/appengine/devices.ex

```diff
@@ -57,6 +103,18 @@ defmodule Astarte.Client.AppEngine.Devices do
     end
   end
 
+  @doc """
+  Returns a values snapshot for a given interface without `:path` option, otherwise
+  returns a value on a given endpoint path.
+
+  ## Options
+
+    * `:path`  - endpoint path
+
+    * `:query` - list of query params
```

Comment:
If the `query` in properties accepts a different set of query parameters w.r.t. datastream, document them here. Otherwise, cross-link to the datastream detailed documentation below.